less/index.less

```diff
@@ -64,113 +64,320 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 10px;
 }
 
-.container {
-    width: 880px;
-    margin: auto;
+.main-content-spacing{
+    display:flex;
+    justify-content:space-around;
 }
 
-/* Header Styles */
+.main-content-colorscheme{
```

Comment:
great mix in.  Also the naming convention is spot on.  It helps people know exactly what they are looking at.